MockedRethinkDB.js

```diff
@@ -0,0 +1,334 @@
+const filter = require("lodash.filter");
+const map = require("lodash.map");
+const flatMap = require("lodash.flatmap");
+const rimraf = require("rmfr");
```

Comment:
I don't think that `rmrf` wraps `rimraf`, it seems to just implement the functionality in a sync manner (https://github.com/mrDarcyMurphy/node-rmrf/blob/master/index.js).

I'd rather a dep like that in the project, can we just use the already-included and `rimraf` library?